web3c/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * objectAssign is an alternative to Object.assign to work with react-native.
+ * Assigns all properties from b onto a.
+ */
+function objectAssign(a, b, enumerable = true) {
```

Comment:
this is a bit of a weird one, in that it seems preferable to have this happen through a babel transpile rather than explicitly not using new features, since we use others that are transpiled back to compatibility